From 1d8d276b3194456ccc0b2bfecac7714aca9047a5 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Tue, 9 Jun 2020 08:42:39 -0700
Subject: [PATCH 08/47] Improve 1890 Hungarian transition.

(Thanks to Michael Deckers.)
* NEWS: Mention this.
* europe (Europe/Budapest): 1890 transition on Nov 1, not Oct.
---
 NEWS   | 3 ++-
 europe | 8 +++++++-
 2 files changed, 9 insertions(+), 2 deletions(-)

diff --git a/NEWS b/NEWS
index 91ab463..e3ab402 100644
--- a/NEWS
+++ b/NEWS
@@ -13,7 +13,8 @@ Unreleased, experimental changes
 
     Correct several transitions for Hungary for 1918/1983.
     For example, the 1983-09-25 fall-back was at 01:00, not 03:00.
-    (Thanks to Géza Nyáry.)
+    (Thanks to Géza Nyáry.)  Also, the 1890 transition to standard
+    time was on 11-01, not 10-01 (thanks to Michael Deckers).
 
   Changes to code
 
diff --git a/europe b/europe
index 48e1eeb..c2a44ad 100644
--- a/europe
+++ b/europe
@@ -1511,6 +1511,12 @@ Zone	Europe/Athens	1:34:52 -	LMT	1895 Sep 14
 			2:00	EU	EE%sT
 
 # Hungary
+
+# From Michael Deckers (2020-06-09):
+# an Austrian encyclopedia of railroads of 1913, online at
+# http://www.zeno.org/Roell-1912/A/Eisenbahnzeit
+# says that the switch [to CET] happened on 1890-11-01.
+
 # From Géza Nyáry (2020-06-07):
 # Data for 1918-1983 are based on the archive database of Library Hungaricana.
 # The dates are collected from original, scanned governmental orders,
@@ -1539,7 +1545,7 @@ Rule	Hungary	1980	only	-	Sep	28	 1:00	0	-
 Rule	Hungary	1981	1983	-	Mar	lastSun	 0:00	1:00	S
 Rule	Hungary	1981	1983	-	Sep	lastSun	 1:00	0	-
 # Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
-Zone	Europe/Budapest	1:16:20 -	LMT	1890 Oct
+Zone	Europe/Budapest	1:16:20 -	LMT	1890 Nov  1
 			1:00	C-Eur	CE%sT	1918
 			1:00	Hungary	CE%sT	1941 Apr  7 23:00
 			1:00	C-Eur	CE%sT	1945
-- 
1.8.3.1

